Bike.java

```diff
@@ -0,0 +1,26 @@
+package com.k7it;
+
+public class Bike {
+
+	int wheel;
+	String name;
+	double power;
+	double engine;
+	
+	public double ccCapacity() {
+		return (power*engine)/wheel;
+	}
+	
+	public static void main (String arg[]) {
+		Bike b1=new Bike();
```

Comment:
object ref name try to use the same name of class name with lower case for example
Bike bike = new Bike();